float32[-100.0, 100.0] ShoulderRot
float32[-100.0, 100.0] ShoulderPitch
float32[-100.0, 100.0] ElbowPitch
float32[-100.0, 100.0] WristPitch
float32[-100.0, 100.0] WristRot